src/orm.ts

```diff
@@ -10,6 +10,7 @@ export class AppDataSource {
   static async initialize() {
     if (!AppDataSource.datasource) {
       const dropSchema = config.get('DROP_DATABASE') === 'true';
+      const synchronize = (config.get('ENVIRONMENT') as string) === 'test';
```

Comment:
Just gotta remember from now on we have to write a migration for a new entity not just write the .ts file.